rules/windows/defense_evasion_suspicious_wmi_script.toml

```diff
@@ -28,7 +28,7 @@ sequence by process.entity_id with maxspan = 2m
 [process where host.os.type == "windows" and event.type == "start" and
    (process.name : "WMIC.exe" or process.pe.original_file_name : "wmic.exe") and
    process.args : ("format*:*", "/format*:*", "*-format*:*") and
-   not process.command_line : "* /format:table *"]
+   not process.command_line : ("* /format:table *", "* /format:table")]
```

Comment:
Am I crazy, or should `* /format:table *` match everything that `* /format:table` matches and more? Or does that specific space character do the trick? If so, wouldn't it make more sense to change it to `* /format:table*`?

Reply:
It doesn't exclude a command line that ends with `/format:table`, but I'm sure if adding that wildcard could enable someone to abuse it by naming the file as table.xsl